client/blocks/post-item/index.jsx

```diff
@@ -71,6 +72,14 @@ class PostItem extends React.Component {
 		return this.inAllSitesModeWithMultipleUsers() || this.inSingleSiteModeWithMultipleUsers();
 	}
 
+	componentDidUpdate( prevProps ) {
+		const { hasExpandedContent } = this.props;
+
+		if ( ! prevProps.hasExpandedContent && hasExpandedContent ) {
+			ReactDom.findDOMNode( this ).scrollIntoView( false );
```

Comment:
We probably want to have something other than `scrollIntoView` here, that scrolls the top of the element to (for example) 100px above the bottom of the viewport.  From the [`scrollIntoView` docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView), it looks like this will scroll the bottom of the element to the bottom of the viewport.

This means the bottom of the share panel _as it is initially rendered_, and the height of the initial render will vary according to the characteristics of the current site and what data is already loaded into the state tree from previous requests.  So I think in order to have consistent behavior here we need to calculate a specific scroll position.